src/common/types.ts

```diff
@@ -1,22 +1,22 @@
-import { ComponentType } from "react";
-
-import { PreloadedQuery, UseQueryLoaderLoadQueryOptions } from "react-relay";
-import { GraphQLTaggedNode, OperationType, VariablesOf } from "relay-runtime";
-
-export type PagePropsWithQuery<T extends OperationType> = {
-    queryReference: PreloadedQuery<T>;
-    loadQuery: (variables: VariablesOf<T>, options?: UseQueryLoaderLoadQueryOptions) => void;
-};
-
-export type QueryInfo = {
-    query: GraphQLTaggedNode;
-    preloadedQuery: PreloadedQuery<OperationType>;
-    queryObject: any;
-};
-
-export type RouteCustom = {
-    path: string;
-    name: string;
-    component: ComponentType<any>;
-    queryInfo?: QueryInfo;
-};
+import { ComponentType } from "react";
```

Comment:
I wonder what is the usage of this file? 
Is it a way of Relay setup?

Reply:
Yes, it's a part of relay setup